src/screens/TopicPageScreen/elements/BottomSheetFollow.tsx

```diff
@@ -51,6 +55,11 @@ const BottomSheetFollow = forwardRef((props: BottomSheetFollowProps, ref: Ref<RB
       try {
         await followTopic(topicName, isIncognito);
         getTopicDetail(topicName);
+        const discoveryInitialTopicResponse = await DiscoveryRepo.fetchInitialDiscoveryTopics();
+        DiscoveryAction.setDiscoveryInitialTopics(
+          discoveryInitialTopicResponse.suggestedTopics as any,
+          discoveryDispatch
+        );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While this code may work as expected, it's important to note that error handling for the `fetchInitialDiscoveryTopics` function is missing. If the fetch operation fails, the application might crash or behave unexpectedly. Consider adding a try-catch block around the fetch operation and handle the error appropriately.
```diff
- const discoveryInitialTopicResponse = await DiscoveryRepo.fetchInitialDiscoveryTopics();
+ let discoveryInitialTopicResponse;
+ try {
+   discoveryInitialTopicResponse = await DiscoveryRepo.fetchInitialDiscoveryTopics();
+ } catch (error) {
+   // Handle error here
+ }
```
Also, using `as any` when setting the initial topics is not recommended because it can lead to runtime errors if the data structure doesn't match what you expect. It would be better to define a proper type for the suggestedTopics.
```diff
- DiscoveryAction.setDiscoveryInitialTopics(
-   discoveryInitialTopicResponse.suggestedTopics as any,
-   discoveryDispatch
- );
+ if (discoveryInitialTopicResponse) {
+   DiscoveryAction.setDiscoveryInitialTopics(
+     discoveryInitialTopicResponse.suggestedTopics as YourDefinedSuggestedTopicsType,
+     discoveryDispatch
+   );
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->